access.go

```diff
@@ -256,13 +256,6 @@ func (s *Server) handleAuthorizationCodeRequest(w *Response, r *http.Request) *A
 
 	// Verify PKCE, if present in the authorization data
 	if len(ret.AuthorizeData.CodeChallenge) > 0 {
-		// https://tools.ietf.org/html/rfc7636#section-4.1
```

Comment:
removing PKCE validation because spec only state that : 
>  If the values are not   equal, an error response indicating "invalid_grant" as described in   Section 5.2 of [RFC6749] MUST be returned.
https://tools.ietf.org/html/rfc7636#section-4.6